selfdrive/car/fw_versions.py

```diff
@@ -242,8 +242,9 @@ def get_fw_versions_ordered(logcan, sendcan, ecu_rx_addrs, timeout=0.1, num_pand
 
     car_fw = get_fw_versions(logcan, sendcan, query_brand=brand, timeout=timeout, num_pandas=num_pandas, debug=debug, progress=progress)
     all_car_fw.extend(car_fw)
-    # Try to match using FW returned from this brand only
-    matches = match_fw_to_car_exact(build_fw_dict(car_fw))
+
+    # If there is a match using this brand's FW alone, finish querying early
+    matches = match_fw_to_car(car_fw, log=False)
     if len(matches) == 1:
```

Comment:
match_fw_to_car always returns a tuple:
```
    ...
    if len(matches):
      return exact_match, matches

  return True, set()
```
So this if statement is irrelevant.

Reply:
ah you're right, good catch!